data/data/bootstrap/bootstrap-in-place/files/opt/openshift/bootstrap-in-place/bootstrap-in-place-post-reboot.sh

```diff
@@ -83,6 +91,7 @@ function clean {
   rm -rf /usr/local/bin/installer-gather.sh
   rm -rf /usr/local/bin/installer-masters-gather.sh
   rm -rf /var/log/log-bundle-bootstrap.tar.gz
+  rm -rf /opt/openshift
```

Comment:
This is another change where the 55a53adb910e commit message points out that this is happening, but doesn't explain why the pivot is useful.

Reply:
I'll update the commit message